bindings/experimental/distrdf/python/DistRDF/Backends/Spark/Backend.py

```diff
@@ -161,3 +168,39 @@ def make_dataframe(self, *args, **kwargs):
         npartitions = kwargs.pop("npartitions", self.optimize_npartitions())
         headnode = HeadNode.get_headnode(npartitions, *args)
         return DataFrame.RDataFrame(headnode, self)
+
+    @staticmethod
+    def RunGraphs(proxies, numthreads=4):
+        """
+        Trigger multiple RDF graphs through multithreading, according to Spark
+        docs on job scheduling at https://spark.apache.org/docs/latest/job-scheduling.html#scheduling-within-an-application.
+
+        Args:
+            proxies(iterable): Action proxies that should be triggered. Only
+                actions belonging to different RDataFrame graphs will be
+                triggered to avoid useless calls.
+
+            numthreads(int, optional): Number of threads to spawn at the same
+                time. Each thread will submit a separate job to the Spark
+                cluster through the same SparkContext. Defaults to 4.
+        """
+
+        # Create queue to store all the action proxies
+        q = queue.Queue()
+
+        for proxy in proxies:
+            q.put(proxy)
+
+        # Function to trigger the computation graph of each proxy in the queue
+        def trigger_loop(queue_):
+            while True:
+                queue_.get().GetValue()
+                queue_.task_done()
+
+        # Create `numthreads` threads that will each submit a Spark job
+        for _ in range(numthreads):
+            worker = threading.Thread(group=None, target=trigger_loop, name=None, args=(q, ), kwargs=None)
```

Comment:
Isn't it a problem that the worker thread objects get all destroyed in this loop (except for the last one)?

Reply:
Not sure, the [Python2 docs](https://docs.python.org/2.7/library/queue.html#Queue.Queue.join) show a similar logic (albeit I guess the `daemon` attribute is playing a non-negligible role in there)

Reply:
Ok for the docs, but these are not daemon, are they?

Reply:
Indeed because I had to remove the keyword argument `daemon` since it's not Python2 compatible. I will change this logic in the next commit also removing the queue part